src/demo/DemoTestRepaymentService.java

```diff
@@ -97,7 +103,8 @@ public static void testGetRepaymentsDueFromTo() throws MambuApiException {
 
 	public static void testUpdateLoanRepaymentsSchedule() throws MambuApiException {
```

Comment:
As per the requirement 4 from MBU-6844 we should also allow adding a new installment on the schedule. On the "if" condition from the line 160 could you please take into account the DYNAMIC_TERM_LOAN loan product type?